src/InfrastructureLayer/Settings/I_Settings.h

```diff
@@ -15,4 +16,5 @@ class I_Settings
     virtual bool customQueueEnable() const = 0;
 
     virtual void setQueueName(QString queueName) = 0;
+    virtual void setDefaultValue() = 0;
```

Comment:
This isn't used anywhere else so it does not need to be on the interface. i.e It can be a private function in Settings.